### DISCUSSION TITLE
Can a volume be mapped for persistence in the docker container?

### DISCUSSION THREAD

**Discussion by 2fst4u** [anse-app/anse#46] [Q&A]
Basically as the title. Is there a volume that can be mapped to maintain persistence across container restarts?

**Comment by yzh990918** [comment]
Sorry, I'm not quite sure what you mean. Are you asking about deploying

**Reply by 2fst4u** [reply]
Are you familiar with how docker containers persist data? If a docker container is restarted and no volumes have been set, all the data is lost and it is like starting fresh.

To keep all your settings like API keys, you need to give the container a volume mapping.
My question is where is the data saved, so that I can map a volume to it.

**Reply by yzh990918** [reply]
IndexDB
